.travis.yml

```diff
@@ -21,7 +21,7 @@ env:
   # Run all tests before db_block_cache_test (db_test, db_test2)
   - JOB_NAME=unittests ROCKSDBTESTS_END=db_block_cache_test
   # Run all tests starting from db_block_cache_test (db_block_cache_test, ..., plain_table_db_test)
-  - JOB_NAME=unittests ROCKSDBTESTS_START=db_block_cache_test ROCKSDBTESTS_END=comparator_db_test
+  - JOB_NAME=unittests MEM_ENV=true ROCKSDBTESTS_START=db_block_cache_test ROCKSDBTESTS_END=comparator_db_test
```

Comment:
I browsed the tests in db_wal_test. I think if we can move DBWALTest.Recover, DBWALTest.PreallocateBlock, DBWALTest.RecoverWithLargeLog, DBWALTest.SyncMultipleLogs, DBWALTest.RecoverWithoutFlush and DBWALTest.RecoverWithoutFlushMultipleCF out of the db_wal_test (for example, go to db_test2.cc, or a new db_wal_test_core.cc), we should be good to go. They are not slow tests.